spec/lib/submission_builder/ty2022/states/return_header_spec.rb

```diff
@@ -61,15 +61,20 @@
         let(:primary_first_name) { "Prim" }
         let(:primary_middle_initial) { "W" }
         let(:primary_last_name) { "Filerton" }
+        let(:primary_suffix) { "jr" }
```

Comment:
after line 88, does it make to add this expectation?
```
expect(doc.at('Filer Primary TaxpayerName NameSuffix').content).to eq("JR")
```
i see the test data is set up but there's no expectation.

Reply:
if i'm missing something please let me know, don't mean to be nit picky. i just didn't see the expectation.